server/models.py

```diff
@@ -283,13 +283,16 @@ class Assignment(Model):
     autograding_key = db.Column(db.String(255))
     uploads_enabled = db.Column(db.Boolean(), nullable=False, default=False)
     upload_info = db.Column(db.Text)
+    published_scores = db.Column(StringList, nullable=False, default=[])
```

Comment:
I'm also open to keeping this as non nullable - not sure what the best thing to do is. I kind of like the idea of having it be null which tells us that the staff haven't touched anything. 

@knrafto thoughts?